src/test/kotlin/org/wycliffeassociates/otter/common/domain/ImportResourceContainerTest.kt

```diff
@@ -55,13 +55,21 @@ class ImportResourceContainerTest {
 
 }
 
-class MockCollectionDao: Dao<Collection> {
-    override fun getById(id: Int): Observable<Collection> {
-        return Observable.just(Mockito.mock(Collection::class.java))
+class MockCollectionDao: ICollectionRepository {
```

Comment:
Just one thing, supposed to be `class MockCollectionRepository` right?

Reply:
Thanks, fixed